docs/developer_setup.md

```diff
@@ -1,142 +1,25 @@
 ## Developer Setup
-Technologies and tools in use:
-- Python: https://www.python.org/
-- Poetry: https://python-poetry.org/
-- Pre-commit: https://pre-commit.com/
-- Docker: https://www.docker.com/
-- FastAPI: https://fastapi.tiangolo.com/
-- Pydantic: https://pydantic-docs.helpmanual.io/
-- SQLAlchemy: https://www.sqlalchemy.org/
-- Alembic: https://alembic.sqlalchemy.org
-
----
-## Python
-CTMS is a Python application. We tend to keep the application up to date with
-the latest version of Python -- we pin to the patch version throughout the
-repository.
-
-To keep your development environment up to date, you may wish to install a
-Python version manager to switch Python versions along with the application.
-Tools for this include:
-- [pyenv](https://github.com/pyenv/pyenv)
-- [asdf](https://asdf-vm.com/)
-
-Dependabot will submit pull requests to update the Python version in the
-Dockerfile, but will miss other places like `pyproject.toml` and Github Action
-workflow files. This `sed`[^1] snippet will find/replace all Python versions in
-one go - in this example, from version `A.B.C` to `X.Y.Z`:
-```bash
-git ls-files | xargs sed -i 's/A\.B\.C/X\.Y\.Z/g'
-```
```

Comment:
We may want to keep this snippet (and the footnote) somewhere ([context](https://github.com/mozilla-it/ctms-api/pull/522#issuecomment-1405795386))

Reply:
ooos